.drone.yml

```diff
@@ -55,7 +55,23 @@ steps:
       event:
         - push
         - tag
-
+  - name: build_push
+    image: plugins/docker
+    settings:
+      repo: rancher/k3d
+      tags:
+        - latest-linux-amd64
+        - "${DRONE_TAG}-linux-amd64"
+      dockerfile: tools/Dockerfile
```

Comment:
Shouldn't this build the Dockerfile in the repo root? :thinking:

Reply:
With the target `dind`? :thinking:

Reply:
Feel free to make the changes if you think they are right! Edits by maintainers are on :)